rules/aep0134/request_unknown_fields.go

```diff
@@ -34,6 +34,7 @@ var unknownFields = &lint.MessageRule{
 		allowedFields := stringset.New(
 			fieldNameFromResource(resource), // AEP-134
 			"allow_missing",                 // AEP-134
+			"path",                          // AEP-134
```

Comment:
can a linter rule be added to make sure path is required? Maybe re-using the get / delete one?

Reply:
Added!